Renderer/DX11/D3D11Renderer.cs

```diff
@@ -82,6 +82,21 @@ public void DrawRectangle(RectangleF rect, Color color, float width = 1.0f)
             this.context.RenderTarget.DrawRectangle(rect, this.brushCache.GetOrCreate(color), width);
         }
 
+        public void DrawFilledRectangle(RectangleF rect, Color color, Color BackgroundColor, uint BorderWidth = 1)
+        {
+            for (uint i = 0; i < BorderWidth; i++)
```

Comment:
move bruchCache get out of the loop, no reason to waste lookup time for each call * i